libs/langgraph/src/graph/messages_annotation.ts

```diff
@@ -88,14 +107,5 @@ export const MessagesAnnotation = Annotation.Root({
  * ```
  */
 export const MessagesZodState = z.object({
```

Comment:
At the end of the day I think we want to discourage using this with Zod 4 going forward